### DISCUSSION TITLE
How to use options to find more than 5 keywords

### DISCUSSION THREAD

**Discussion by CedYF** [retextjs/retext#80] [General]
const file = await retext(options)
          .use(retextPos) 
          .use(retextKeywords)
          .process(txt)
          
          I have this code but I would like to find 10 keywords instead of 5. But I can't figure out how to enable multiple keywords? Can anyone help?

**Comment by wooorm** [comment]
Pass options: https://github.com/retextjs/retext-keywords#options.

```js
.use(retextKeywords, {maximum: 123}
```
